copy-migrations.sh

```
@@ -5,7 +5,6 @@ echo "Copying prisma migration files..."
 MIGRATIONS_TARGET=${CONFIGURATION_BUILD_DIR}/${UNLOCALIZED_RESOURCES_FOLDER_PATH}
 
 rm -rf "$MIGRATIONS_TARGET/migrations"
-mkdir "$MIGRATIONS_TARGET/migrations"
-cp -r ${SRCROOT}/../migrations ${MIGRATIONS_TARGET}
+cp -r ${SRCROOT}/../prisma/migrations "${MIGRATIONS_TARGET}/migrations"
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)